contents/node/nodejs.md

```diff
@@ -0,0 +1,203 @@
+# Introduction to Node.js
+
+Author: Rachael Sim
+
+# Overview
+
+*Node.js* is a JavaScript runtime built on Chrome’s V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient.
```

Comment:
Does this chapter assumes the reader is familiar with JS? if so, mention that. If not, the chapter can be read by a wider audience (e.g., to motivate people to learn JS in order to use a cool tool like node.js) but then you need to tweak the text accordingly to avoid alienating reading who are not familiar with JS.